pages/resources/channel.mdx

```diff
@@ -1353,20 +1353,20 @@ Note that when sending a message, you must provide a value for **at least one of
-| activity?                   | [message activity](/resources/message#message-activity-object) object                | The rich presence activity to invite users to                                                                                              |
-| application_id?             | snowflake                                                                            | The application ID of the activity to create a rich presence invite for (defaults to the primary activity if unspecified)                  |
-| flags?                      | integer                                                                              | The [message's flags](/resources/message#message-flags) (only `SUPPRESS_EMBEDS`, `SUPPRESS_NOTIFICATIONS`, and `VOICE_MESSAGE` can be set) |
-| files[n]? ^1^               | file contents                                                                        | Contents of the file being sent (max 10)                                                                                                   |
-| payload_json? ^1^           | string                                                                               | JSON-encoded body of non-file params                                                                                                       |
-| attachments? ^1^            | array[partial [attachment](/resources/message#attachment-object) object]             | Partial attachment objects with `filename` and `description` (max 10)                                                                      |
+| Field             | Type                                                                      | Description                                                                                                                                |
+| ----------------- | ------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| content?          | string                                                                    | The message contents (max 2000 characters)                                                                                                 |
+| embeds? ^2^       | array[[embed](/resources/message#embed-object) object]                    | Embedded `rich` content (max 6000 characters, max 10)                                                                                      |
+| ~~embed? ^2^~~    | ~~[embed](/resources/message#embed-object) object~~                       | ~~Embedded `rich` content (max 6000 characters)~~, deprecated in favor of `embeds`                                                         |
```

Comment:
does this no longer work?

Reply:
Consistency between how deprecated is displayed in other tables. If that isn't desired I can revert it.